packages/node_modules/sample-browser-auth-implicit/app.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env browser */
+/* global ciscospark */
+/* eslint-disable camelcase */
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable no-alert */
+/* eslint-disable no-console */
+/* eslint-disable require-jsdoc */
+
+// Declare some globals that we'll need throughout
+let spark;
```

Comment:
For the purposes fo this example, we really want to show how `spark` takes care of auth automatically, so let's actually try and avoid using the authenticate and checkAuthentication functions.

up here, just do `let spark = ciscospark.init()`, then inside the ready event, wire up the button and status ui.